pkg/kafka/cluster_admin_client_mock_impl.go

```diff
@@ -26,13 +26,26 @@ const (
 	defaultMockControllerID = 1
 )
 
-var (
-	// defaultMaxMessageBytes specifies the default max message bytes.
-	defaultMaxMessageBytes = "10485760"
-	// defaultMaxMessageBytes specifies the default min insync replicas for broker and default topic.
+const (
+	// defaultBrokerMessageMaxBytes specifies the broker's `message.max.bytes`, default to 1048576.
+	defaultBrokerMessageMaxBytes = "1048576"
+
+	// defaultTopicMaxMessageBytes specifies the topic's `max.message.bytes`, default to 1048576.
+	defaultTopicMaxMessageBytes = "1048576"
+
+	// defaultMinInsyncReplicas specifies the default min insync replicas for broker and default topic.
 	defaultMinInsyncReplicas = "1"
 )
 
+var (
+	// BrokerMessageMaxBytes is the broker's `message.max.bytes`
+	BrokerMessageMaxBytes = defaultBrokerMessageMaxBytes
```

Comment:
I think we really only need to define these values once?

Reply:
what do you mean the `once` ?

`BrokerMessageMaxBytes` is exposed, then can be modified by other pieces of code.

Reply:
Yes, then we don't need to repeat the definition twice. Just define it once directly?

Reply:
I don't think it's defined twice. 

`BrokerMessageMaxBytes` is different from `defaultBrokerMessageMaxBytes`.
`BrokerMessageMaxBytes` is different from `TopicMaxMessageBytes`.

Reply:
I mean BrokerMessageMaxBytes and defaultBrokerMessageMaxBytes we only need one?

Reply:
I think they have different meaning. `BrokerMessageMaxBytes` is exposed to be configurable, `defaultBrokerMessageMaxBytes` is the default value.

If we only keep one, `defaultBrokerMessageMaxBytes` would be removed, and then `BrokerMessageMaxBytes = 1048576`

Reply:
Yes, please delete it, it makes no sense.